.github/workflows/auto-label-pr.yml

```
@@ -0,0 +1,65 @@
+                await github.rest.issues.addLabels({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  issue_number: context.issue.number,
+                  labels: [labelName]
+                });
+                console.log('Label added successfully');
+              }
+            } catch (error) {
+              console.error('Error managing label:', error);
+              core.setFailed(`Failed to add label: ${error.message}`);
+            }
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)